lib/number-checks.js

```diff
@@ -0,0 +1,15 @@
+// To support Node v.0.10.x
+var number = module.exports = {
+  MAX_SAFE_INTEGER: Number.MAX_SAFE_INTEGER || 9007199254740991,
```

Comment:
I think it doesn't really matter. MAX_SAFE_INTEGER is a well-known constant and as such it's ok to represent it as a static number value. No need to re-compute it all the time.